website/content/en/status/report-2024-10-2024-12/ports-collection-accessibility-colors.adoc

```diff
@@ -0,0 +1,23 @@
+=== Ports Collection Accessibility - Colors Low Vision
+
+Link: +
+link:https://wiki.freebsd.org/portconfig[Project wiki page] URL: link:https://wiki.freebsd.org/portconfig[]
+
+Contact: FreeBSD Accessibility mailing list <freebsd-accessibility@FreeBSD.org> +
+Contact: Alfonso Sabato Siciliano <asiciliano@FreeBSD.org>
+
+FreeBSD provides the Ports Collection to give users and administrators a simple way to install applications.
+The collection provides tens of thousands of ports; port configuration is a key feature.
+It is possible to configure a port before the building and installation.
+The command "make config" uses a text user interface (TUI) to set up port options interactively.
+
+Recently low vision users (mainly with cataracts) have requested new features to easily change the colors of the TUI.
+Several features have been implemented to allow changing colors, for example: a new environment variable to set the UI to black and white, or the ability to set colors by reading a configuration file at runtime.
+All features have been described in the https://man.freebsd.org/portconfig/1[portconfig(8)] manual since version 0.6.2.
```

Comment:
Suggestion: "All features have been described in man:portconfig[1] since version 0.6.2."

Reply:
Fixed Thank you!